src/Data/Audience.elm

```diff
@@ -5920,6 +5963,17 @@ audiencesJSON =
                 "shared": true,
                 "category": null,
                 "folder": null
+            },
+            {
+                "id": 91958,
+                "name": "My Folder doesn't exist",
+                "expression": {
+                },
+                "curated": null,
+                "type": "shared",
+                "shared": true,
+                "category": null,
+                "folder": 9999999
```

Comment:
Nice fixture, we should bake this permanently into the original challenge data 👍